app/controllers/api/topics_controller.rb

```diff
@@ -0,0 +1,7 @@
+module Api
+  class TopicsController < ApplicationController
+    def index
+      @topics = Topic.all
```

Comment:
I think for now is fine like this. But keep in mind that usually pagination is preferred instead of getting all the topic at once (we could have thousands of topics with their images that will be bring to memory in this case). But as I said, given that this is an onboarding project, is good as is 😄